.drone/drone.jsonnet

```diff
@@ -318,6 +318,9 @@ local manifest(apps) = pipeline('manifest') {
     steps: [
       make('check-generated-files', container=false) { depends_on: ['clone'] },
       make('test', container=false) { depends_on: ['clone', 'check-generated-files'] },
+      run('clone-main', commands=['cd ..', 'git clone $CI_REPO_REMOTE loki-main', 'cd -']),
+      run('test-main', commands=['cd ../loki-main', 'BUILD_IN_CONTAINER=false make test']) { depends_on: ['clone-main', 'check-generated-files'] },
```

Comment:
Curious about the dependency on `check-generated-files` here. Why do we need this?

Reply:
I was pattern matching, I'll remove it.

Reply:
Looks like test doesn't need it either I removed that dependency, too